examples/post-sync/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build stage
+ARG GOLANG_IMAGE=golang:1.24
+
+FROM ${GOLANG_IMAGE} as builder
+
+WORKDIR /
```

Comment:
Copying to the root of a container is generally a bad idea. It can have a lot of possible side effects when copying into an existing linux boot file structure. It's often better to make a /workdir/ or /workspace/ or something to copy into and build in, to avoid side effects.

Reply:
Added WORKDIR